research-institute-for-nature-and-forest.csl

```diff
@@ -7,63 +7,95 @@
     <link href="http://www.zotero.org/styles/research-institute-for-nature-and-forest" rel="self"/>
     <author>
       <name>Maarten Stevens</name>
-      <uri>http://www.mendeley.com/profiles/maarten-stevens/</uri>
+      <uri>https://www.mendeley.com/profiles/maarten-stevens/</uri>
     </author>
-    <contributor>
-      <name>Thierry Onkelinx</name>
-      <uri>http://www.mendeley.com/profiles/thierry-onkelinx/</uri>
-    </contributor>
-    <contributor>
+    <author>
       <name>Floris Vanderhaeghe</name>
       <email>floris.vanderhaeghe@inbo.be</email>
+    </author>
+    <contributor>
+      <name>Thierry Onkelinx</name>
+      <uri>https://www.mendeley.com/profiles/thierry-onkelinx/</uri>
     </contributor>
     <category citation-format="author-date"/>
     <category field="biology"/>
-    <updated>2019-08-01T00:00:00</updated>
+    <updated>2020-08-06T00:00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
     <terms>
       <term name="in">In</term>
+      <term name="editor" form="short">
```

Comment:
This isn't necessary to add: it's the default

Reply:
45e86d65e